### PR TITLE
Use eredis with Unix domain sockets support

### PR DESCRIPTION
**Pull request by igors**
This PR addresses #2041

Proposed changes include:
* Point to my fork of eredis with unix domain sockets support
* update documentation with usage example

### CODE REVIEW

**Comment by kanes115** [comment]
Unix domain sockets change [link](https://github.com/igors/eredis/commit/e9688a1ec98e10acca310f19266bd95f976bc262)

File: doc/modules/mod_global_distrib.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ Global distribution modules expose several per-datacenter metrics that can be us
 
 #### Redis session storage options
 
-* **server** (string, default: `"127.0.0.1"`): Address of the Redis listener.
+* **server** (string, default: `"127.0.0.1"`): Address of the Redis listener. For Unix domain sockets use `{local, "/path/to/listener.sock"}` (available starting with OTP 19).
```
Comment (fenek) [inline doc/modules/mod_global_distrib.md]: `(available starting with OTP 19)` is not necessary, as MongooseIM doesn't support OTP 18 or older. :)

Comment (igors) [reply doc/modules/mod_global_distrib.md]: Fixed - thanks for noticing.

**Comment by fenek** [comment]
@michalwski 

I think we should verify our dependencies for activity, as there might be more than just `eredis`, which could be worth forking.

**Pull request merged**